wandb/sdk/wandb_artifacts.py

```diff
@@ -289,11 +287,11 @@ def name(self) -> str:
         return self._name
 
     @property
-    def full_name(self) -> str:
+    def qualified_name(self) -> str:
```

Comment:
are you ok with breaking backward compatibility? 
this property doesn't seem to be private -> assuming we have users that used it -> this change could break them. You could maybe add depreciation message and suggest using the new property.

Reply:
This was added on April 14 ([here](https://github.com/wandb/wandb/pull/5314)) and isn't in the docs yet. So it should be relatively okay to rename.

Reply:
yeah, if it makes sense for the artifact team, i don't have issues with this. 
Just maybe better practice for the future to be more careful about api changes between releases 😄